app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,65 @@
+class ExpensesController < ApplicationController
+
+  require 'csv'
+  
+  def index
+    @tour = Tour.find(params[:id])
+    @expenses = Expense.where(tour_id:params[:id])
+
+  end
+
+  def new
+    @tour = Tour.find(params[:id])
+    @expense = Expense.new
+  end
+
+  def create
```

Comment:
Do you want to keep this route for now?

Reply:
Will keep for now - no need to reference the route